app/controllers/cart_products_controller.rb

```diff
@@ -0,0 +1,21 @@
+class CartProductsController < ApplicationController
+  def create
```

Comment:
This controller action is where the business logic for adding an item to a cart currently lives.